backend/model/db.js

```diff
@@ -1,14 +1,31 @@
 const Sequelize = require('sequelize')
 const UserModel = require('./users')
-const PostModel = require('./posts')
+const ProjectModel = require('./projects')
+const UserProjectModel = require('./userProjects')
+const ProjectTagModel = require('./projectTags')
+const ProjectCollaboratorModel = require('./projectCollaborators')
+const ProjectFileModel = require('./projectFiles')
 
 const sequelize = new Sequelize('akademise', 'root', 'password', {
   host: 'ec2-54-173-244-46.compute-1.amazonaws.com',
   dialect: 'mysql',
 })
 
+
 const User = UserModel(sequelize, Sequelize)
-const Post = PostModel(sequelize, Sequelize)
+const Project = ProjectModel(sequelize, Sequelize)
+const UserProject = UserProjectModel(sequelize,Sequelize)
+const ProjectTag = ProjectTagModel(sequelize, Sequelize)
+const ProjectCollaborator = ProjectCollaboratorModel(sequelize, Sequelize)
+const ProjectFile = ProjectFileModel(sequelize, Sequelize)
+
+Project.hasMany(ProjectTag, {foreignKey : 'project_id' , onDelete: 'CASCADE' })
+Project.hasMany(ProjectCollaborator, {foreignKey : 'project_id' , onDelete: 'CASCADE' })
+UserProject.belongsTo(Project, {foreignKey : 'project_id', onDelete: 'CASCADE'})
+ProjectCollaborator.belongsTo(User, {foreignKey: 'user_id'})
```

Comment:
I think we might also have "onDelete: 'CASCADE'" on ProjectCollaborator user_id.
The design choice should be decided, will the program be able to delete users from DB and will associated Collaboration deleted from ProjectCollaborator table?
```suggestion
ProjectCollaborator.belongsTo(User, {foreignKey: 'user_id'})
```

Reply:
We talked over this issue, currently it is not necessary.

Reply:
i think its not necessary for the first milestone, we could decide it later. It functions well right now